README.md

```diff
@@ -350,35 +298,29 @@ This architecture consists of the following helm chart installs:
       --set userArtifacts.image=coherence-demo-sidecar:3.0.0-SNAPSHOT \
       coherence/coherence
    ```
-   
-   > **Note:** By default, the latest version of the chart will be used.  You can add `--version 1.0.0` to 
-   > choose a specific version. E.g. 1.0.0 in the above case.
-   
-   You can also choose a specific version of the Coherence Docker image by specifying the following in the
-   above `helm install` command:
-   
+   > **Note:** By default, the latest version of the chart is used. You can add `--version 1.0.0` to select a specific version. For example, 1.0.0 is used in the command.
+
+   You can also choose a specific version of the Coherence Docker image by specifying the following in the `helm install` command:
+
    ```bash
    --set coherence.image=container-registry.oracle.com/middleware/coherence:12.2.1.4.0
    ```
-
    Because we use stateful sets, the coherence cluster will start one pod at a time.
```

Comment:
This line is no longer true so can be removed "Because we use stateful sets, the coherence cluster will start one pod at a time."